compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/fbt/error.todo-fbt-param-with-unicode.expect.md

```diff
@@ -0,0 +1,30 @@
+
+## Input
+
+```javascript
+import fbt from "fbt";
+
+function Component(props) {
+  const element = (
+    <fbt desc={"Dialog to show to user"}>
+      Hello <fbt:param name="user name ☺">{props.name}</fbt:param>
+    </fbt>
+  );
+  return element.toString();
+}
+
+export const FIXTURE_ENTRYPOINT = {
+  fn: Component,
+  params: [{ name: "Jason" }],
+};
+
+```
+
+
+## Error
+
+```
+Property arguments[0] of CallExpression expected node to be of a type ["Expression","SpreadElement","JSXNamespacedName","ArgumentPlaceholder"] but instead got "JSXExpressionContainer"
```

Comment:
This error is not caught and handled by Forget as it occurs in `babel-plugin-fbt`. Going to go back to hack in a check in `MemoizeFbtAndMacroOperandsInSameScope` for now, but ideally we would have some type / annotation on `Identifier` instances to know which strings are fbt operands